articles/azure-databricks/quickstart-create-databricks-workspace-portal.md

```diff
@@ -70,40 +70,39 @@ In this section, you create an Azure Databricks workspace using the Azure portal
     Accept all other default values other than the following:
 
     * Enter a name for the cluster.
-    * For this article, create a cluster with **4.0** runtime. 
+    * For this article, create a cluster with **4.0** runtime.
     * Make sure you select the **Terminate after \_\_ minutes of inactivity** checkbox. Provide a duration (in minutes) to terminate the cluster, if the cluster is not being used.
     
-    Select **Create cluster**. Once the cluster is running, you can attach notebooks to the cluster and run Spark jobs. 
+    Select **Create cluster**. Once the cluster is running, you can attach notebooks to the cluster and run Spark jobs.
 
 For more information on creating clusters, see [Create a Spark cluster in Azure Databricks](https://docs.azuredatabricks.net/user-guide/clusters/create.html).
 
 
 ## Download a sample data file
 Download a sample JSON data file and save it into Azure blob storage.
 
-1. Download this sample JSON data file [from Github](https://raw.githubusercontent.com/Azure/usql/master/Examples/Samples/Data/json/radiowebsite/small_radio_json.json) onto your local computer. Right-click and save as to save the raw file locally. 
+1. Download this sample JSON data file [from Github](https://raw.githubusercontent.com/Azure/usql/master/Examples/Samples/Data/json/radiowebsite/small_radio_json.json) onto your local computer. Right-click and save as to save the raw file locally.
 
-2. If you don't already have a storage account, create one. 
-   - In the Azure portal, select **Create a resource**.  Select the **Storage** category, and select **Storage Accounts**  
-   - Provide a unique name for the storage account.
-   - Select **Account Kind**: **Blob Storage**
-   - Select a **Resource Group** name. Use the same resource group you created the Databricks workspace.
-   
-   For more information, see [Create an Azure Blob storage account](../storage/common/storage-quickstart-create-account.md). 
+2. If you don't already have a storage account, create one.
+    - In the Azure portal, select **Create a resource**. Select the **Storage** category, and select **Storage Accounts**
```

Comment:
We should use 3 spaces for indenting bullets. 4 spaces will render the bullets as code I believe.